src/DeviceInterface/WindowsInterface.js

```diff
@@ -78,15 +78,15 @@ export class WindowsInterface extends InterfacePrototype {
                     break
                 }
                 default: {
-                    if (this.globalConfig.isDDGTestMode) {
+                    if (this.isTestMode()) {
```

Comment:
Nothing to see, just updating to use the abstraction instead of the property directly.